src/System Application/Test/EntityText/src/MktTextRedTeamAIT.Codeunit.al

```diff
@@ -0,0 +1,156 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+
+namespace System.Test.EntityText;
+
+using System.TestTools.AITestToolkit;
+using System.Text;
+
+codeunit 134641 "Mkt Text RedTeam AIT"
+{
+    Subtype = Test;
+
+    [Test]
+    procedure TaglineTest()
+    var
+        AITestContext: Codeunit "AIT Test Context";
+        EntityTextCod: Codeunit "Entity Text";
+        Facts: Dictionary of [Text, Text];
+        Format: Enum "Entity Text Format";
+        InputJson: JsonObject;
+        InputToken: JsonToken;
+        InputPrompt: Text;
+        Response: Text;
+    begin
+        InputJson.ReadFrom(AITestContext.GetInput().ValueAsText());
+        InputJson.Get('inputPrompt', InputToken);
+        InputPrompt := InputToken.AsValue().AsText();
+
+        Format := Enum::"Entity Text Format"::Tagline;
+        AddFacts(Facts, InputPrompt, Format);
+        Response := EntityTextCod.GenerateText(Facts, Enum::"Entity Text Tone"::Inspiring, Format, Enum::"Entity Text Emphasis"::None);
+        AITestContext.SetTestOutput(PrepareOutput(BuildFactsText(Facts), Response, Enum::"Entity Text Tone"::Inspiring, Format));
+    end;
+
+    [Test]
+    procedure TaglineParagraphTest()
+    var
+        AITestContext: Codeunit "AIT Test Context";
+        EntityTextCod: Codeunit "Entity Text";
+        Facts: Dictionary of [Text, Text];
+        Format: Enum "Entity Text Format";
+        InputJson: JsonObject;
+        InputToken: JsonToken;
+        InputPrompt: Text;
+        Response: Text;
+    begin
+        InputJson.ReadFrom(AITestContext.GetInput().ValueAsText());
```

Comment:
The properties can be directly read using AITTestContext.GetInput() -> (returns Codeunit "Test Input Json")

AITestContext.GetInput().Element('inputPrompt').ValueAsText()

Can we use 'question' in the dataset instead of 'inputPrompt'? This would standardize the term usage across all the datasets.

And then you can also use AITestContext.GetQuestion().ValueAsText() to read the input.